pyproject.toml

```diff
@@ -38,6 +38,8 @@ dependencies = [
     "scipy>=1.13.1",
     "seaborn>=0.13.2",
     "statsmodels>=0.14.2",
+    "mpl-image-compare",
```

Comment:
This should be in the `optional-dependencies` `dev` section.

Reply:
ah I was going off the [`pyproject.toml`](https://github.com/IMMM-SFA/msd_uncertainty_ebook/blob/dev/pyproject.toml#L67) in dev which uses a separate `test` `optional-dependencies` section and it looks like you're using `dev`. So when I installed `pip install -e ".[test]"` it was failing because pytest wasn't installed.